### ISSUE TITLE
Syntax error on import pretty_midi

### ISSUE THREAD

**Issue by bzamecnik** [craffel/pretty-midi#113]
```
>>> import pretty_midi

  File "/Users/bzamecnik/anaconda/lib/python3.4/site-packages/pretty_midi/utilities.py", line 84
    ur'^(?P<key>[ABCDEFGabcdefg])'
                                 ^
SyntaxError: invalid syntax
```

Version: pretty-midi==0.2.6. Python 3.4.5

The original code is:

```
pattern = re.compile(
        # Start with any of A-G, a-g
        '^(?P<key>[ABCDEFGabcdefg])'
```

So it seems that `six` modifies the code and adds the `u` (unicode string) prefix which should be in Python 2 but not Python 3. But why?

**Comment by craffel** [comment]
Duplicate of #111.  This was fixed in 66e428a.  You'll need to use bleeding-edge until I make another release.

**Comment by bzamecnik** [comment]
Thanks.
